Tasks/AppCenterDistributeV1/appcenterdistribute.ts

```diff
@@ -167,6 +167,23 @@ function publishRelease(apiServer: string, releaseUrl: string, releaseNotes: str
         ]
     };
 
+    const branchName = process.env['BUILD_SOURCEBRANCHNAME'];
+    const sourceVersion = process.env['BUILD_SOURCEVERSION'];
+
+    // Builds staretd by App Center has the commit message set
+    const commitMessage = process.env['LASTCOMMITMESSAGE'];
+
+    // Including these information for distribution notification to have additional context
+    if (branchName && sourceVersion && commitMessage) {
```

Comment:
Could `commitMessage` be null or empty? If so, we should remove it from the if condition.

Reply:
Do you want to show that the rest of the information (branch name and commit hash) in the notification, when there is no commit message? If a customer uses this task in VSTS (without App Center), then commit message would be null, where branch name and source version are likely available

Reply:
Okay - updated